### DISCUSSION TITLE
Passwordless authentication

### DISCUSSION THREAD

**Discussion by danieldunderfelt** [payloadcms/payload#751] [Feature Requests & Ideas]
Hi!

I am evaluating Payload as a backend for a content-focused mobile app (built with React-native), but I don't want to use password authentication. I'd much prefer if the user simply enters their email, gets sent a code, and enters the code in the app. SMS would be good as well, but my use case depends on the user having an email address from specific domains.

**Comment by DanRibbens** [comment]
Hey @danieldunderfelt,

Great timing on asking about this. If you read the release notes from [0.19.2](https://github.com/payloadcms/payload/compare/v0.19.1...v0.19.2) you can see we have moved away from forcing email/passwords in as a default to allow other auth strategies. This goes back to the extensible roadmap item https://github.com/payloadcms/payload/discussions/290.

Now that this is technically possible, we will finalize the documentation around extending auth. We anticpate many Payload plugins for all kinds of auth strategies coming out shortly.

**Comment by GermanJablo** [comment]
Closing as complete, for the same reasons Dan mentioned. You can read our documentation on Auth at https://payloadcms.com/docs/authentication/overview.
